### DISCUSSION TITLE
Cherry and Sniffer themed consoles.

### DISCUSSION THREAD

**Discussion by MagmaSlime123** [eccentricdevotion/TARDIS#720] [Ideas]
Title.

**Comment by eccentricdevotion** [comment]
Budget with cherry wood walls.
How would you make a sniffer themed console?

**Reply by MagmaSlime123** [reply]
Probably a lush cave with sniffer eggs and different plants grown from the seeds it picks up.

Basically the cave console but with some extra edits.
